test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,207 @@
+" days"
+" ether"
+" finney"
+" hours"
+" minutes"
+" seconds"
+" szabo"
+" weeks"
+" wei"
+" years"
+"!="
+"%"
+"&"
+"("
+")"
+"*"
+"**"
+"+"
+"++"
+"-"
+"--"
+".balance"
+".call("
+".callcode("
+".creationCode"
+".delegatecall( "
+".gas("
+".kill("
+".length "
+".now"
+".push("
+".runtimeCode"
+".send("
+".staticcall("
+".transfer("
+".value("
+"/"
+"//"
+"0**0"
+"1.1"
+"2e10"
+":="
+";"
+"<"
+"<<"
+"<="
+"=="
+">"
+">="
+">>"
+"CALLCODE "
+"DELEGATECALL "
+"[a, b, c]"
+"\\udead"
+"\\xff"
+"^"
+"abi.encode("
+"abi.encodePacked("
+"abi.encodeWithSelector("
+"abi.encodeWithSignature("
+"add("
+"addmod("
+"anonymous"
+"assembly {"
+"assert("
+"block.blockhash("
+"block.coinbase("
+"block.difficulty("
+"block.gaslimit("
+"block.number("
+"block.timestamp("
+"blockhash("
+"bool "
+"byte "
+"bytes("
+"bytes1 "
+"bytes10 "
+"bytes11 "
+"bytes12 "
+"bytes13 "
+"bytes14 "
+"bytes15 "
+"bytes16 "
+"bytes17 "
+"bytes18 "
+"bytes19 "
+"bytes2 "
+"bytes20 "
+"bytes21 "
+"bytes22 "
+"bytes23 "
+"bytes24 "
+"bytes25 "
+"bytes26 "
+"bytes27 "
+"bytes28 "
+"bytes29 "
+"bytes3 "
+"bytes30 "
+"bytes32 "
+"bytes4 "
+"bytes5 "
+"bytes6 "
+"bytes7 "
+"bytes8 "
+"bytes9 "
+"console.log("
+"constant "
+"constructor "
+"continue"
+"contract "
+"delete "
+"do"
+"ecrecover("
+"else"
+"emit a("
+"enum B {"
+"event HighestBidIncreased(address bidder, uint amount"
+"event"
+"external "
+"false "
+"fixed "
+"fixed128x128 "
+"for (a=0;a<2;a++)"
+"for"
+"function bid() public payable {"
+"gasleft("
+"hex"001122FF""
+"hex\"001122FF\""
+"if"
+"int "
+"int x = -2**255;"
+"int256 "
+"int8 "
+"interface i {"
+"internal "
+"is"
+"keccak256("
+"let x := "
+"library l {"
+"log0("
+"log1("
+"log2("
+"log3("
+"log4("
+"mapping("
+"memory m ="
+"modifier onlySeller() {"
+"msg.data("
+"msg.gas("
+"msg.sender("
+"msg.sig("
+"msg.value"
+"mulmod("
+"new "
+"payable "
+"pop "
+"pragma experimental ABIEncoderV2;"
+"pragma experimental SMTChecker;"
+"pragma solidity >=0.4.0 <0.6.0;"
+"pragma solidity ^90.90.0"
+"public "
+"pure "
+"require("
+"require(msg.sender == 0,\"\""
+"return "
+"return"
+"returns "
+"revert("
+"ripemd160("
+"self"
+"selfdestruct("
+"sha256("
+"sha3("
+"storage sto = "
+"string str = "
```

Comment:
This will always result in a parser error, you either need `string memory str = ` or `string storage str = `.

Reply:
done. added both `string memory str =` and `string storage str=` since generated code probably differs between them.